modules/openapi-generator/src/main/resources/typescript-inversify/api.service.mustache

```diff
@@ -152,7 +152,12 @@ export class {{classname}} {
 
 {{#hasFormParams}}
         let formData: FormData = new FormData();
+        {{#isMultipart}}
+        headers['Content-Type'] = 'multipart/form-data';
```

Comment:
It should‘t be set explicitly, as there would be a problem with the separator, or am I missing something?

Reply:
Yes that’s technically correct, it shouldn’t be explicitly set. But because of the way the HttpClient does an Object.assign to get some default headers, if you leave content-type undefined it becomes application/json which is still wrong

So alongside setting this content-type, I added a check to HttpClient to see if it is a multipart upload and do a `delete headers[“content-type”]`

Reply:
ok, I see. maybe
https://github.com/OpenAPITools/openapi-generator/blob/07d145828f33e9e2d9e5be6694bfb1d29db4fe07/modules/openapi-generator/src/main/resources/typescript-inversify/HttpClient.mustache#L46-L51

should be fixed instead? @gualtierim what is your opinion?